ui/package.json

```diff
@@ -138,7 +140,7 @@
     "@types/google.analytics": "0.0.39",
     "@types/jasmine": "^3.3.5",
     "@types/jasminewd2": "^2.0.3",
-    "@types/jest": "^23.3.11",
+    "@types/jest": "^26.0.20",
```

Comment:
I hit some issues when trying to upgrade jest (very annoying issues that only showed up in certain situations on a local machine: https://precisionmedicineinitiative.atlassian.net/browse/RW-5959). Is this upgrade necessary with the change in typescript version? Or is there another reason to bump this?

If we upgrade this, we should verify that this issue does not show again locally - I can track down the exact repro if needed